packages/cli/src/commands/deploy/index.ts

```diff
@@ -42,9 +43,13 @@ export default async client => {
 
   let { localConfig } = client;
   if (!localConfig || localConfig instanceof Error) {
-    localConfig = readLocalConfig(paths[0]);
+    const maybeConfig = readLocalConfig(paths[0]);
+    if (maybeConfig) {
+      localConfig = maybeConfig;
+    }
   }
-  const stats = {};
+
+  const stats: any = {};
```

Comment:
This shouldn't be `any`. Although it might be removed in PR #6388 soon.

Reply:
I see that the PR is merged, but how can I solve this now in my PR ? cc @TooTallNate  @styfle